src/main/java/com/iexec/worker/result/ResultService.java

```diff
@@ -308,16 +308,14 @@ public String getIexecUploadToken() {
             final String expectedDomainName = "iExec Result Repository";
             final String actualDomainName = domain.getName();
             if (!Objects.equals(actualDomainName, expectedDomainName)) {
-                log.error("Domain name does not match expected name" +
-                                " [expected:{}, actual:{}]",
+                log.error("Domain name does not match expected name [expected:{}, actual:{}]",
```

Comment:
Do you prefer to have placeholders on the same line than static text? I find it more readable to have all parameters on a single line, without anything else.

Reply:
Do you mean you prefer `"[expected:{}, actual:{}]", value1, value2` ?
I tend to prefer having the string on a single line to avoid missing spaces which impair readability.

Reply:
Some hints about line breaks: 
- https://github.com/spring-projects/spring-framework/wiki/Code-Style#line-wrapping
- https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping
- https://blog.sebastian-daschner.com/entries/operator-newline-formatting
(doesn't give a full answer)

Reply:
I mean I prefer the way it was before. Something like:
```java
log.info("Some static text that needs precision",
  " [param1:{}; param2: {}]",
  param1, param2);
```

We may miss a space sometimes but I find it usually easier to read as data is separated where it has to be (three parts: static text, dynamic text & vars).

----
Regarding @jeremyjams' links: the first one and the second one differ on where to break a line when there's an operator - here, the `+` operator. There's another interesting part on the maximum number of characters on a line which should be around 90... Which doesn't help much there :')

Reply:
Fun fact : The 90 characters reference is an old legacy coming from first terminals and typewriters where you had around 80 characters. At this point, you had to perform a carriage return or a line feed (still typewriter references by the way) ^_^

As mentioned, there is no absolute answer.
I tend to prefer single line logs to force myself to have concise and quickly readable messages.
My eyes work best on a single line and are a bit slower for multi line comments.

There is small caveat with your approach @mcornaton , isolating the part encapsulated in `[]` produces redundancy code smells in sonar, which would require to introduce a variable and would produce less readable statements from my point of view.

For the last time, those are preferences, and I totally understand that several people will have different preferences.

Reply:
We'll have times to discuss on how to write logging events in our code.
I won't touch this for the moment.